addOns/jython/src/main/java/org/zaproxy/zap/extension/jython/JythonOptionsPanel.java

```diff
@@ -114,7 +114,7 @@ public void validateParam(Object object) throws Exception {
             if (!modulesPath.exists()) {
                 throw new NoSuchFileException(
                         Constant.messages.getString(
-                                "jython.options.error.modulepath.notexist", modulesPath));
+                                "jython.options.error.modulepath.nonexistent", modulesPath));
```

Comment:
Same as https://github.com/zaproxy/zap-extensions/pull/4306#discussion_r1050796248

Reply:
Similarly, `nonexistent` is a word, whereas the phrase would really be `does not exist` with punctuation of sorts.

Again, no attachment to it, but, just why a spelling set will include it.

Reply:
Thanks for clarifying josh, I think the thing here was more that the key name was changed but not the associated text, which seemed a bit incongruent.